### PR TITLE
Card component initial add

### PR DESCRIPTION
**Pull request by thusithak**
Fixes Issue [#90](https://github.com/wso2/ux-theme-wso2/issues/90)

### CODE REVIEW

File: module/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (dakshika) [inline module/components/Card/Card.js]: use the other pattern.

Comment (dakshika) [reply module/components/Card/Card.js]: import React, {Compo..} from 'react';

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Card
+
+.card{
+  padding: $card-spacer-y $card-spacer-x;
+  min-height: $card-min-height;
+}
+.card-raised{
```
Comment (jeradrutnam) [inline module/stylesheets/components/_card.scss]: `.car-raised` should be in `.card` class as is should be as `.card .card-raised`

File: module/stylesheets/components/_card.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  background: $card-bg;
+  box-shadow: $btn-box-shadow;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_card.scss]: Shall we create new variable for $card-box-shadow and use $btn-box-shadow there ?

File: module/components/Card/Readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Card:
+
+```js
+<Card raised></Card>
```
Comment (jeradrutnam) [inline module/components/Card/Readme.md]: If there are no children markup should be `<Card raised />`

**Review by jeradrutnam — CHANGES_REQUESTED** [review]
Please see my inline comments.

File: module/stylesheets/components/_card.scss (hunk 3 above)
Comment (thusithak) [reply module/stylesheets/components/_card.scss]: Done

**Review by dakshika — APPROVED** [review]

File: module/components/Card/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+const defaultProps = {
+    raised: false,
+};
+/**
+ * Card component class
+ * @returns Card
+ */
+export class Card extends Component {
+    render() {
```
Comment (dakshika) [inline module/components/Card/Card.jsx]: Add jsDocs

File: module/components/Card/CardHeader.jsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+const defaultProps = {
+    className: 'card-header',
+};
+/**
+ * Extended CardHeader class
+ * @extend {ReactComponent} ReactstrapCardHeader
+ */
+export class CardHeader extends Component {
+    render() {
```
Comment (dakshika) [inline module/components/Card/CardHeader.jsx]: Add JsDocs

**Comment by dakshika** [comment]
Add component  to index

**Review by dakshika — CHANGES_REQUESTED** [review]
Add component to index

File: module/components/Card/Card.jsx (hunk 5 above)
Comment (thusithak) [reply module/components/Card/Card.jsx]: Done

File: module/components/Card/CardHeader.jsx (hunk 6 above)
Comment (thusithak) [reply module/components/Card/CardHeader.jsx]: Done

File: module/components/Card/Card.js (hunk 1 above)
Comment (thusithak) [reply module/components/Card/Card.js]: Done

**Review by dakshika — APPROVED** [review]

**Review by jeradrutnam — APPROVED** [review]
Approved

**Comment by thusithak** [comment]
Also Fixes Issue #104

**Pull request merged**